<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PokedexPage</class>
 <widget class="QDialog" name="PokedexPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>774</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pokedex Info</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>-600</x>
     <y>-110</y>
     <width>1961</width>
     <height>1021</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/resources/resources/pokedexBackground.jpg</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>181</width>
     <height>781</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="name">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="attack">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>Attack</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="health">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>Health</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="speed">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>Speed</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="type">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="secondType">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>Second type</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="description">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>471</width>
     <height>771</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="nameText">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="attackText">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="healthText">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="speedText">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="typeText">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="secondTypeText">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="descriptionText">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>1009</x>
     <y>560</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.972, y2:0.909, stop:0 rgba(220, 0, 0, 253), stop:0.301676 rgba(147, 0, 0, 254), stop:0.653631 rgba(131, 0, 0, 254), stop:1 rgba(0, 0, 0, 255));

font: 11pt &quot;Fixedsys&quot;;
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
